src/minuteman_ct_latency_observer.erl

```diff
@@ -104,7 +99,7 @@ start_link() ->
 init([]) ->
   process_flag(min_heap_size, 2000000),
   minuteman_vip_events:add_sup_handler(fun push_vips/1),
-  ets:new(connection_timers, [{keypos, #ct_timer.id}, named_table]),
+  ets:new(connection_timers, [{keypos, #ct_timer.id}, named_table, public, {read_concurrency, true}]),
```

Comment:
Is this public only for the test? I don't see If so, we should make it a testing fixture.